pkg/tests/ossm-federation/testdata/traffic-splitting/west-mesh/smcp.yaml

```diff
@@ -89,6 +92,8 @@ spec:
                 memory: 128Mi
               limits: {}
   security:
+    identity:
```

Comment:
This should only be added when the test case es executed in ROSA as far I know

Reply:
No, this matches what the test did before the refactoring. It always set `security.identity.type`. See here: https://github.com/maistra/maistra-test-tool/blob/maistra-2.4/testdata/examples/federation/install.sh#L27

Reply:
Let's discuss whether we should use this for all environments or not. The tests definitely set this for all environments, but I remember us having an issue with one of the environments.

Reply:
Ignore this, single cluster needs the variable in ROSA is not needed. I check the previous changes and comments and that security property is needed for SingleCluster